tools/clang/unittests/HLSL/Objects.cpp

```diff
@@ -919,6 +861,9 @@ TEST_F(ObjectTest, TemplateArgConstraints) {
   }
 }
 
+// The test that requires this function is pending complete
+// support for primitive types
+#if 0
```

Comment:
The code enclosed herein is only used by code similarly ifdefed out with this same comment.